docs/learn/smart-contract-developers/create-lsp7-token.md

```diff
@@ -186,26 +227,31 @@ Deploying with an EOA is more straightforward, but you miss on the Universal Pro
 
 ```ts title="scripts/deployEOA.ts"
 import { ethers } from 'hardhat';
-import * as dotenv from 'dotenv';
-
-dotenv.config();
 
 async function main() {
-  // Hardhat has some issues with EIP 1559 settings, so you need to force it
-  // See this issue for more info: https://github.com/NomicFoundation/hardhat/issues/3418
-  const { maxFeePerGas, maxPriorityFeePerGas } =
-    await ethers.provider.getFeeData();
-
   const customToken = await ethers.getContractFactory('CustomToken');
 
-  const Token = await customToken.deploy({
-    maxFeePerGas,
-    maxPriorityFeePerGas,
-    type: 2,
-  });
+  // custom token constructor params
+  const tokenName = 'My Custom Token';
+  const tokenSymbol = 'MCT';
+  const tokenContractOwnerAddress = (await ethers.getSigners())[0].address;
```

Comment:
can this await fail here ?

also maybe : 
  const tokenContractOwnerAddress = (await ethers.getSigners())[0]?.address;